lib/index.js

```diff
@@ -32,12 +32,15 @@ function buildConfig(pluginConfig, assetManager, { watcher, browsers, compact })
 			selectedBrowsers = browsers.defaults;
 		}
 
+		let { fingerprint } = bundleConfig;
```

Comment:
I thought that I can't merge this with the `let { browserslist } = bundleConfig` line, because that's one unit (from the whitespace around it) that only handles the browser config.

Reply:
Misunderstood and adjusted.